### DISCUSSION TITLE
Sus Lack of Progress - On verge of breakthrough

### DISCUSSION THREAD

**Discussion by nbollman** [mudler/LocalAI#1213] [General]
It feels like this application has trailed off development just as the program was becoming useful.  Dying to see a Mistral update... feels like Im viewing a archived version of the page or something. I see pieces of mistral...
(base) nbollman@TheWallWindow:~/ai/LocalAI$ curl http://localhost:8080/models/available | jq '.[] | select(.name | contains("mistral")) | {name: .name, urls: .urls}' | jq -r '.name + " " + (.urls | join(" "))'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
(base) nbollman@TheWallWindow:~/ai/LocalAI$
 | {name: .name, urls: .urls}' | jq -r '.name + " " + (.urls | join(" "))'
1tuanpham__instruct_en-vi_12000_1e_thebloke_mistralic-7b-1-gptq_lora_causal_lm__instruct_en-vi_12000_1e__adapter_model.bin https://huggingface.co/1TuanPham/Instruct_en-vi_12000_1e_TheBloke_Mistralic-7B-1-GPTQ_LORA_CAUSAL_LM
1tuanpham__instruct_en-vi_7500_1epoch_thebloke_mistralic-7b-1-gptq_lora_causal_lm__adapter_model.bin https://huggingface.co/1TuanPham/Instruct_en-vi_7500_1epoch_TheBloke_Mistralic-7B-1-GPTQ_LORA_CAUSAL_LM
1tuanpham__instruct_en-vi_7500_1epoch_thebloke_mistralic-7b-1-gptq_lora_causal_lm__instruct_en-vi_7500_1epoch__adapter_model.bin https://huggingface.co/1TuanPham/Instruct_en-vi_7500_1epoch_TheBloke_Mistralic-7B-1-GPTQ_LORA_CAUSAL_LM
1tuanpham__instruct_en-vi_8000_1e_thebloke_mistralic-7b-1-gptq_lora_causal_lm__adapter_model.bin https://huggingface.co/1TuanPham/Instruct_en-vi_8000_1e_TheBloke_Mistralic-7B-1-GPTQ_LORA_CAUSAL_LM

**Comment by lunamidori5** [comment]
I know that they are working on a full backend rework right now, So stay tuned for the breakthrough!

**Comment by mudler** [comment]
@nbollman it is a community driven project - so you are welcome in helping out. I'm adding now an example on how to run mistral manually here: https://github.com/go-skynet/LocalAI/pull/1214
